### DISCUSSION TITLE
What is the current state of the Identity Hub?

### DISCUSSION THREAD

**Discussion by maximilianschmidt-zf** [eclipse-edc/IdentityHub#40] [General]
Hello together,

we have taken a look at the identity hub and are interested in it's current state and future development.

The Identity Foundation has released a 0.0.1 draft spec for the identity hub.
https://identity.foundation/decentralized-web-node/spec/0.0.1-predraft/

Our first question would be, whether it is planned to implement this spec and future publications from the Identity Foundation.

And our second question would be, if it is planned to implement additional DID methods. We are currently especially interested in DID Indy.

And the last question would be is there some kind of milestone planning for the project when additional features will be added? 
Or maybe is this an MVP project for the MVD and it is currently not planned to invest more resources to it?

Thanks in advance and Kind regards!

**Comment by jimmarino** [comment]
Hi @maximilianschmidt-zf 

Our goal is to add support for https://identity.foundation/decentralized-web-node/spec/, which is the updated version of the old Identity Hub spec.

We can discuss support for Indy as a DID method, although this may be orthogonal to identity hubs.

**Reply by paullatzelsperger** [reply]
as for planning: your best chance is to follow the milestone planning in EDC, which is the parent project. Anything related to IH will likely be discussed there. We have currently no plans to setup a similar planning here as well.
